MM/5.10.0/config/media_manager_5/labels/Download.dcl

```diff
@@ -638,3 +638,18 @@ resource configservice_label download_main_window_download_button {
   ]
 }
 
+resource configservice_label download_main_window_not_recommended_quality {
+  key = 'DOWNLOAD_MAIN_WINDOW_NOT_RECOMMENDED_QUALITY'
+  group = 'Download'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'This version is not recommended for use as it may require stretching the asset to a larger size, resulting in loss of quality.'
```

Comment:
```suggestion
      default_translation = 'This quality is not recommended for use as it may require stretching the asset to a larger size, resulting in loss of quality.'
```